branding/src/palette.ts

```diff
@@ -6,7 +6,20 @@ All rights reserved.
 This code is licensed under the BSD-3 license found in the LICENSE file in the root directory of this source tree and at https://opensource.org/licenses/BSD-3-Clause.
 **/
 
-export const blue = {
+type BrandingColorType = {
```

Comment:
Can we unify BrandingColorType and UIColorType with a single definition (just make the missing fields optional ones)...ideally they are the same type.